rust/stackablectl/CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Add pre-built binary for `aarch64-unknown-linux-gnu` ([#232]).
+
+[#232]: https://github.com/stackabletech/stackable-cockpit/pull/232
```

Comment:
Nit: Normally we group all links for a link at the bottom (instead of having one list for added and one for changed

Reply:
Ah yeah... I will fix that. Does it maybe make sense to split them up? Especially when we have to deal with an increasing amount of entries in the changelog?

Reply:
Changed in 834a91a